tests/config/no-options/config.php

```diff
@@ -0,0 +1,8 @@
+<?php
```

Comment:
This is not needed. The whole point of being able to load a configuration from an array was, precisely, to avoid having a fixed directory structure.